aptible/resource_environment.go

```diff
@@ -47,9 +50,41 @@ func resourceEnvironment() *schema.Resource {
 func resourceEnvironmentCreate(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
 	client := meta.(*aptible.Client)
 	handle := d.Get("handle").(string)
-	orgID := d.Get("org_id").(string)
 	stackID := int64(d.Get("stack_id").(int))
 
+	// there are a few  scenarios where org_id can be fully gathered from the data provided above
```

Comment:
Extra space in comment

```suggestion
	// there are a few scenarios where org_id can be fully gathered from the data provided above
```